dymos/grid_refinement/hp_adaptive/hp_adaptive.py

```diff
@@ -499,7 +506,7 @@ def refine(self, refine_results, iter_number):
                         err_val = np.dot(np.absolute(b - b_hat).ravel(),
                                          np.array([2 ** l for l in range(seg_size[gd.transcription])])) / beta
 
-                        if err_val > phase.refine_options['tolerance'] and merge_seg[k + 1]:
+                        if err_val > phase.refine_options['tolerance']/10 and merge_seg[k + 1]:
```

Comment:
Is this division by 10 intentional? (i.e., is it part of a defined formula in refinement?)  Otherwise, why not just reduce the default tolerance by an order of magnitude?

Reply:
Yeah it's intentional. It's only done in cases where we're reducing the grid order or merging segments. So we want to hold it to a tighter tolerance so that we don't get the algorithm going in circles with merging and splitting

Reply:
Thanks, makes sense.